vertica/tests/test_unit.py

```diff
@@ -35,6 +35,25 @@ def test_ssl_config_ok(aggregator, tls_instance):
     aggregator.assert_service_check("vertica.can_connect", status=AgentCheck.OK, tags=['db:abc', 'foo:bar'])
 
 
+def test_ssl_legacy_config_ok(aggregator, tls_instance_legacy):
```

Comment:
Made this test to ensure that the legacy TLS config remapping and logic worked correctly.